SwagLyricsBot/general_commands.py

```diff
@@ -11,9 +11,12 @@
 
 class GeneralCommands(commands.Cog, name="General"):
 
-    def __init__(self, bot, session):
+    def __init__(self, bot, session, current=None):
+        if current is None:
+            current = {}
```

Comment:
Why not make `current` an empty dict by default?

Reply:
because dicts are mutable, so it will persist across inits. https://docs.python-guide.org/writing/gotchas/